### DISCUSSION TITLE
Can you create a github pages subdomain with the project instead of username

### DISCUSSION THREAD

**Discussion by ianmbloom** [community/community#22178] [Other Feature Feedback, Questions, & Ideas]
<p>Is it possible to create a github pages subdomain with the project instead of username as in project_name.github.io</p>

**Comment by HendricoMaximus** [comment]
<p>I do not think so. It only the basic tutorial for creating one, <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#types-of-github-pages-sites" rel="noopener nofollow ugc">located here</a>, says that you have to use <code>username.github.io</code>.</p>

**Comment by ianmbloom** [comment]
<p>Ok thank you for the reply.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>You can create an organisation using the project name (if it’s available), and then build the pages repo there. You’ll be able to use the <code>organisation_name.github.io</code> pages URL then.</p>
<p>Alternately, you can also assign a custom domain if you own one.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site" target="_blank" rel="noopener">Configuring a custom domain for your GitHub Pages site - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site" target="_blank" rel="noopener">//docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
